app/src/main/java/org/shadowice/flocke/andotp/View/EntryViewHolder.java

```diff
@@ -251,7 +251,7 @@ public void updateValues(Entry entry) {
 
         int thumbnailSize = settings.getThumbnailSize();
         if(settings.getThumbnailVisible()) {
-            thumbnailImg.setImageBitmap(EntryThumbnail.getThumbnailGraphic(context, entry.getIssuer(), entry.getLabel(), thumbnailSize, entry.getThumbnail()));
+            thumbnailImg.setImageBitmap(EntryThumbnail.getThumbnailGraphic(context, TextUtils.isEmpty(issuerText) ? "": issuerText, entry.getLabel(), thumbnailSize, entry.getThumbnail()));
```

Comment:
We should probably fix the `EntryThumbnail.getThumbnailGraphic()` function to properly handle `null` values instead.